pyvista/core/utilities/transformations.py

```diff
@@ -392,3 +394,176 @@ def rotation(
         translate_from_origin[:3, 3] = valid_point
 
         return translate_from_origin @ rotate @ translate_to_origin
+
+
+def decompose(
```

Comment:
Could rename this to `decomposition` to make it more consistent with other names in this module

Reply:
Renamed in https://github.com/pyvista/pyvista/pull/6628/commits/fbba4ebe240dbb99fef10577d226ac1a94a5c7ef